parameter/DefaultElementLibrary.h

```diff
@@ -46,7 +47,13 @@ class CDefaultElementLibrary: public CElementLibrary
 {
 public:
 
-    explicit CDefaultElementLibrary(bool bEnableDefaultMechanism = true);
+    /**
+     * Class constructor
+     *
+     * @param[in] logger the logger provided by the client
+     * @param[in] bEnableDefaultMechanism boolean allowing to activate VirtualSubsystem use
+     */
+    explicit CDefaultElementLibrary(core::log::ILogger& logger, bool bEnableDefaultMechanism = true);
```

Comment:
you're right it has no effect. I forgot to remove it, but as you said, it can be good to keep it anyway